demos/poisson/poisson_mixed.py.rst

```diff
@@ -144,6 +144,18 @@ visualisation tool such as `ParaView <http://www.paraview.org/>`__ ::
 
   File("poisson_mixed.pvd").write(u)
 
+We could use the built in plot function of firedrake by calling 
+:func:`plot <firedrake.plot.plot>` to plot a surface graph. Before that,
+matplotlib.pyplot should be installed and imported::
+
+  import matplotlib.pyplot as plt
+  plot(u, 1)
```

Comment:
I think this demonstrates that it would be nice if the number of sampling points could be determined from the element degree...